run_dir/static/css/status.css

```diff
@@ -40,13 +40,13 @@ multiple places and typically control typography, or other common elements.
 	overflow: hidden;
 }
 
-#project_quotas td {
+.table-disk-quotas td {
```

Comment:
I wasn't even sure that all this css is needed. When I removed it completely, it didn't work. So I just decided to keep everything, just to be sure